columbus-5/swagger.fixed.yaml

```diff
@@ -39955,7 +39985,7 @@ paths:
       parameters:
         - name: proposal_status
           type: string
-          enum: *PROPOSAL_STATUSES
+          enum: *PROPOSAL_STATUSES_2
```

Comment:
seems like we don't need the old proposal statuses anymore cause API handles them as invalid:
```json
{
  "code": 3,
  "message": "strconv.ParseInt: parsing \"PROPOSAL_STATUS_UNSPECIFIED\": invalid syntax",
  "details": []
}
```

and since it's the only endpoint using the enum, it would be nice to not keep obsolete stuff. so please just replace the PROPOSAL_STATUSES string values with digits

Reply:
I think PROPOSAL_STATUSES used in responses, and they are really behaving this way, so we can't delete them

Reply:
But I can rename PROPOSAL_STATUSES_2 -> PROPOSAL_STATUSES_INPUT

Reply:
aw shit. sounds ok then